src/tests/functional/shared_test_classes/include/shared_test_classes/single_op/comparison.hpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+
+#include <map>
+
+#include "gtest/gtest.h"
+#include "shared_test_classes/base/ov_subgraph.hpp"
+#include "common_test_utils/test_constants.hpp"
+#include "ngraph_functions/utils/ngraph_helpers.hpp"
+
+namespace ov {
+namespace test {
+
+typedef std::tuple<
+    std::vector<InputShape>,             // Input shapes tuple
+    ov::element::Type,                   // NG Inputs precision
+    ngraph::helpers::ComparisonTypes,    // Comparison op type
+    ngraph::helpers::InputLayerType,     // Second input type
+    ov::element::Type,                   // IE in precision
+    ov::element::Type,                   // IE out precision
```

Comment:
Please check the check related to output precision. Could it be refined, or depends on In type

Reply:
Now there is only one inType. And outType = inType

Reply:
I strongly suggest to not mix input/output types with model type - inType/OutType are used to set up `PreProcProcessing` over `Model`. So, if only one type is expected, let's leave it as Model Type

Reply:
In most tests they are the same.
In tests where they are not, we can use another approach

Reply:
Agree. Changed